acoular/sdinput.py

```diff
@@ -90,15 +90,19 @@ def _set_sample_freq(self, f):
         self._sample_freq = f
 
     def device_properties(self):
-        """Returns
+        """
+        Display the properties of the sounddevice input device.
+
+        Returns
         -------
         Dictionary of device properties according to sounddevice
         """
         return sd.query_devices(self.device)
 
     def result(self, num):
-        """Python generator that yields the output block-wise. Use at least a
-        block-size of one ring cache block.
+        """Python generator that yields the output block-wise.
```

Comment:
```suggestion
        """
        Python generator that yields the output block-wise.
```

Again `GL01 Docstring text (summary) should start in the line immediately after the opening quotes (not in the same line, or leaving a blank line in between)`